tests/ghbug335.phpt

```diff
@@ -0,0 +1,30 @@
+--TEST--
+GH Bug #335: APCu stampede protection is broken
+--SKIPIF--
+<?php require_once(dirname(__FILE__) . '/skipif.inc'); ?>
```

Comment:
This also needs something like `if (!extension_loaded('pcntl') die('skip pcntl required');`, as pcntl is an optional extension (and iirc not available on Windows).